Tools/Scripts/run-javascriptcore-tests

```diff
@@ -555,8 +558,8 @@ if (defined $remoteConfigFile) {
 if (defined $archs) {
     die "$archs not supported by the provided binary, which supports '$archsInBuild'" if index($archsInBuild, $archs) == -1;
 } else {
-    # Fallback is x86_64, which we replace with the native architecture if the native architecture is in the provided build
-    $archs = "x86_64";
+    # Fallback is the architecture of the host machine
```

Comment:
This comment says what the code below does, not why it does that. Please add some discussion of how arm64 vs. arm64e is handled in this case. We always have problems with that. 

Also, please add a period at the end of the comment.

Reply:
arm64e is handled here https://github.com/WebKit/WebKit/blob/main/Tools/Scripts/run-javascriptcore-tests#L564 (just below my changes). I've added a note in this comment about using `/usr/bin/arch` in favour of `lipo` to avoid the Xcode dependency.